pkg/authorization/registry/clusterpolicy/strategy.go

```diff
@@ -47,14 +47,12 @@ func (strategy) PrepareForUpdate(obj, old runtime.Object) {
 	_ = obj.(*authorizationapi.ClusterPolicy)
 }
 
-// Validate validates a new policy.
 func (strategy) Validate(ctx kapi.Context, obj runtime.Object) fielderrors.ValidationErrorList {
-	return validation.ValidateClusterPolicy(obj.(*authorizationapi.ClusterPolicy))
+	return validation.Validator.Validate(obj)
```

Comment:
> Why make this change?

It made it easy to determine the status of name of validation for all of our kinds: https://github.com/openshift/origin/issues/1120.  Rather than having to know about the name validation function and objectmeta used by every kind, its possible to simply require the validation in one place.  It also makes the reviews safer: "does your strategy use the normal validator?  Ok, you won't break anything egregious".  It also allows unit tests to be sure that no kind is added without validation and it makes it impossible to screw up the rootScopedResources in the RESTMapper.

I can't think of an easy alternative that does all those things, but I guess I could inspect every validation method we have and hope nobody screws it up after the fact.